tests/acceptance.suite.dist.yml

```diff
@@ -15,15 +15,15 @@ modules:
         WPDb:
             dsn: 'mysql:host=127.0.0.1;dbname=wpgraphql_test'
             user: 'root'
-            password: ''
+            password: 'mysql'
```

Comment:
instead of modifying this file, you can copy it to `acceptance.suite.yml` and it will be `.gitignored` so you can use your local creds without versioning them